js/plugins/api.js

```diff
@@ -181,7 +181,8 @@ export class Api {
             credentials: 'include',
             headers: Object.assign({}, DEFAULT_HEADERS, WRITE_HEADERS, headers || {}),
         })
-        .catch(this.onError);
+        .catch(this.onError)
```

Comment:
yes we do: https://github.com/opendatateam/udata/blob/dev/js/plugins/api.js#L96-L102 ;)
In fact, fetch simplify error handling with [`response.ok`](https://developer.mozilla.org/fr/docs/Web/API/Response/ok). (I think we already talked about it :wink:)